client/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>my fine proxy</title>
+  </head>
+  <body>
+    <div id="descriptions"></div>
+    <br>
+    <div id="Tiers"></div>
+    <br>
+    <div id="individual-items"></div>
+    <br>
+    <div id="bonuses"></div>
+    <!-- <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
+    <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script> -->
+    <script src="http://localhost:3663/bundle.js"></script>
+    <script src="http://localhost:3031/bundle.js"></script>
+    <script src="http://localhost:3101/static/js/bundle.js"></script>
+    <script src="http://localhost:3200/bundle.js"></script>
```

Comment:
I routed these a little differently using my scripts to point to my proxy endpoints and then writing get requests for each route in my `server/index.js`. I have no idea what best practice is or the pros and cons of either. Sharing information.

Reply:
I saw. I think the way you did it is interesting, but I find mine more readable at this stage, rather than looking at a bunch of routes. I may switch to your method though. Will have to look into the best practices.